### ISSUE TITLE
Incorrect timestamp of blocks mined

### ISSUE THREAD

**Issue by cameronrll** [trufflesuite/ganache#268]
Depending on which page you look at the "mined on" info on, it changes. On the "Blocks" tab it displays the correct time, however if you click on the block in the list it will always show <hour>:01:00

![image](https://user-images.githubusercontent.com/6898996/34901974-508928da-f80a-11e7-8e3a-02e8eb5a7e0b.png)

![image](https://user-images.githubusercontent.com/6898996/34901982-6304f6c4-f80a-11e7-8ab6-3e2550cb277c.png)

![image](https://user-images.githubusercontent.com/6898996/34901994-7c65eab0-f80a-11e7-90f9-d758253a36a6.png)

![image](https://user-images.githubusercontent.com/6898996/34901999-8b37d3dc-f80a-11e7-9818-9819e2061f9c.png)

## Expected Behavior
Correct timestamp to be shown on both screens

## Current Behavior
One correct, one wrong.

## Possible Solution
Looks like it is just a bug in the ui, that is only correctly displaying the hour, but forcing the rest to be <hour>:01:00

issue is likely here.

MiniBlockCard.js (how mined on is formatted)
```
            <div className="RowItem">
              <div className="MinedOn">
                <div className="Label">MINED ON</div>
                <div className="Value">
                  <Moment unix format="YYYY-MM-DD HH:mm:ss">
                    {block.timestamp}
                  </Moment>
                </div>
              </div>
            </div>
```

BlockCard.js (where issue is)
```
       <div className="Label">MINED ON</div>
              <div className="Value">
                <Moment unix format="YYYY-MM-DD HH:MM:SS">
                  {block.timestamp}
                </Moment>
              </div>
       </div>
```

notice the difference
 - YYYY-MM-DD HH:mm:ss (correct formatting)
 - YYYY-MM-DD HH:MM:SS (incorrect formatting)

## Steps to Reproduce (for bugs)
1. Cause blocks to be created near the hour change (e.g create 1st block at 2:59:00 and 2nd at 3:04:00)
2. Look at the blocks on the block tab, and note the correct time
3. Click on 1st block and notice how the time is now 2:01:00
4. Click on 2nd block and notice how the time is now 3:01:00

## Your Environment
* Version used: v1.0.2
* Operating System and version: Ubuntu 16.0

**Comment by cameronrll** [comment]
I have fixed this issue, but sadly it seems I can't submit the branch to create a PR due to access reasons.

**Comment by seesemichaelj** [comment]
Fixed in #269
